app/api/export/index.js

```diff
@@ -0,0 +1,135 @@
+import moment from 'moment';
+import { sendFileToUser } from './utils';
+
+export type TransactionExportRow = {
+  type: 'in' | 'out',
+  amount: string,
+  fee: string,
+  date: Date
+}
+
+export type CsvData = {
+  headers: Array<string>,
+  rows: Array<Array<string>>
+}
+
+export type TransactionExportFileType = 'csv';
+export type TransactionExportDataFormat = 'CoinTracking';
+
+export type ExportFileResponse = {
+  data: Blob,
+  fileType: TransactionExportFileType
+}
+
+/**
+ * This api layer provides access to the electron local storage
```

Comment:
electron -> browser? :D

Reply:
Damned copy-paste 😬 Fixed the whole comment )